controllers/users.js

```diff
@@ -0,0 +1,10 @@
+const { send } = require("express/lib/response")
```

Comment:
This can go! Pretty sure it'll be an auto-import that's wormed it's way in. If I were you, I'd include it's removal in the PR for your next ticket.